src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.img_settings {
+  align-self: center;
+  justify-content: center;
+  height: 40px;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.nav__wrap {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: block;
+}
+
+.body_header_div_nav_ul {
```

Comment:
OMG! why so complicated?) maybe just make it an element of the nav block and chose something like nav__list?